EventKit/eventkit-cloud
# Scale Workers for Default Tasks

## Description of Improvement

Currently default system tasks like provider checks, cleaning up queues, and other related tasks are being picked up by workers that are meant for processing datapacks.  This PR changes that so that anything put onto the default celery queue gets picked up by it's own workers.  This assures that user's datapacks won't get caught behind other tasks.

## How to test

Pull down the PR and bring your containers down and then back up.  You may want to change the time on provider checks for example to have them run every 1 minute instead of every 30 minutes so you aren't waiting too long.  You should see celery containers get spun up when the provider checks (or any other default task) get put on the queue.  You should also see that these tasks are not picked up by workers meant for datapacks.

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
